src/pretix/plugins/paypal2/templates/pretixplugins/paypal2/checkout_payment_confirm.html

```diff
@@ -2,10 +2,25 @@
 
 <p>
```

Comment:
`<p>` surrounding div und h2 does not make sense. We need to remove the closing </p> as well and add it to the `{% else %}` code path further down.
```suggestion
```